compiler/README.md

```diff
@@ -1,55 +1,156 @@
 gRPC Kotlin Codegen Plugin for Protobuf Compiler
-==============================================
+================================================
 
 This generates the Kotlin interfaces out of the service definition from a
 `.proto` file. It works with the Protobuf Compiler (`protoc`).
 
-Normally you don't need to compile the codegen by yourself, since pre-compiled
-binaries for common platforms are available on Maven Central:
+### Build Tool Plugins
 
-1. Navigate to https://mvnrepository.com/artifact/io.grpc/protoc-gen-grpc-kotlin
-2. Select a package version
-3. Click "Files: View All"
+Usually this compiler is used via a built took plugin, like in Gradle, Maven, etc.
 
-However, if the pre-compiled binaries are not compatible with your system,
-you may want to build your own codegen.
+For Gradle include the [protobuf plugin](https://github.com/google/protobuf-gradle-plugin) with at least version `0.8.13`, like:
```

Comment:
nit: add comma: `For Gradle, include...`

Reply:
```suggestion
For Gradle, include the [protobuf plugin](https://github.com/google/protobuf-gradle-plugin) with at least version `0.8.13`, like:
```